mirrord/layer/src/file/filter.rs

```diff
@@ -99,6 +99,9 @@ fn generate_remote_ro_set() -> RegexSet {
         // AWS cli cache
         // \.aws\/cli\/cache\/.+\.json
         r".aws/cli/cache/.+\.json$",
```

Comment:
Since the function on longer only handles the aws file, could you please move the line about AWS from the function's docs to near the AWS line?